src/prommis/uky/uky_flowsheet.py

```diff
@@ -220,14 +218,25 @@ def main():
 
     scaled_model = set_scaling(m)
 
-    assert_units_consistent(scaled_model)
-    assert degrees_of_freedom(scaled_model) == 0
+    if degrees_of_freedom(scaled_model) != 0:
+        raise AssertionError(
+            "The degrees of freedom are not equal to 0."
+            "Check that the expected variables are fixed and unfixed."
+            "For more guidance, run assert_no_structural_warnings from the IDAES DiagnosticToolbox "
+        )
 
     initialize_system(scaled_model)
 
+    solve(scaled_model)
+
+    fix_organic_recycle(scaled_model)
```

Comment:
A comment is probably need here to clarify what this is doing, as you normally do not fix a recycle stream. I suspect this might be a case of a manual tear stream where you did not create an `Arc`, but you should explain that here.